test/JDBC/expected/TestSpatialPoint-vu-prepare.out

```diff
@@ -338,15 +397,29 @@ CREATE VIEW TextFromGeog AS SELECT STAsText(location) AS TextRepresentation FROM
 
 CREATE VIEW BinaryFromGeog AS SELECT STAsBinary(location) AS BinaryRepresentation FROM SPATIALPOINTGEOG_dt;
 
-CREATE VIEW CoordsFromGeog AS SELECT long(location), lat(location)  AS Coordinates FROM SPATIALPOINTGEOG_dt;
+CREATE VIEW CoordsFromGeog AS SELECT Long(location), Lat(location)  AS Coordinates FROM SPATIALPOINTGEOG_dt;
+
+CREATE VIEW GeogView AS SELECT location.Long, location.Lat AS Coordinates FROM SPATIALPOINTGEOG_dt;
```

Comment:
Let's have a test case where there is a subquery.STX syntax is used inside a view.

Reply:
Done